internal/commands/app/create.go

```diff
@@ -335,16 +330,18 @@ func createFromTemplate(realmClient realm.Client, appID, groupID, templateID, ba
 		s.Start()
 		defer s.Stop()
 
-		zipPkg, err := realmClient.ClientTemplate(
+		zipPkg, ok, err := realmClient.ClientTemplate(
 			groupID,
 			appID,
 			templateID,
 		)
 		if err != nil {
 			return err
 		}
-
-		if err := local.WriteZip(path.Join(rootDir, frontendPath, templateID), zipPkg); err != nil {
+		if !ok {
+			return fmt.Errorf("template '%s' does not have a frontend to pull", templateID)
+		}
+		if err := local.WriteZip(path.Join(rootDir, local.FrontendPath), zipPkg); err != nil {
```

Comment:
we still dropped `templateID` from this filepath (hopefully a test needs updating here too)